lib/main.js

```diff
@@ -0,0 +1,103 @@
+'use babel'
+
+import { CompositeDisposable } from 'atom';
+import { Linterhub, LinterhubTypes } from 'linterhub-ide';
+import { Integration } from './integration';
+import * as path from 'path';
+import LinterhubStatusView from './views/status';
+import LinterhubManagerView from './views/manager';
+
+let subscriptions
+let statusBar
+let managerView
+let linter
+
+export default {
+  config: {
```

Comment:
You can move this to a `configSchema` key in your `package.json`, allowing Atom to determine your settings without needing to parse/transpile and load your code.